### DISCUSSION TITLE
Is it possible to have postal on the same server as postfix

### DISCUSSION THREAD

**Discussion by k040711** [postalserver/postal#2704] [Help with using Postal]
hi
Is it possible to have postal installed on the same server I have a postfix with smtp installed?

I'm curious if they'll be a port issue since they're both mail apps. (example port 25)

so example i got user@example.com able to login to like roundcube and send and receive email through webmail and also have postal on the same server sending bulk email to the mailing list from the same domain example.com...

thanks.

**Comment by willpower232** [comment]
You definitely can't have multiple programs on the same port on the same IP address so even if you do resolve that, at some point you will probably find yourself confused when email ends up in the wrong place. At the very least it would be good to get a second IP address on the server so you can split the traffic easily.

**Reply by k040711** [reply]
ok understood, 

I'm wondering then how do I take advantage of the MX capability of postal, is there some kind of an interface I can install to be able to access / see / reply to the webmail thats coming back?  like roundcube or horde mail as an example

tnx

**Reply by willpower232** [reply]
Postal has no concept of mailboxes, POP, or IMAP, so all the messages would need to be cached by whatever is sending them.

Alternatively, you can have a look at routing to forward messages back to yourself or webhooks for more general alerts

https://docs.postalserver.io/other/auto-responders-and-bounces
https://docs.postalserver.io/developer/webhooks
